utils/node_local_test.py

```diff
@@ -6395,10 +6395,26 @@ def run(wf, args):
     # If the perf-check option is given then re-start everything without much
     # debugging enabled and run some micro-benchmarks to give numbers for use
     # as a comparison against other builds.
+    run_fi = False
+
     if args.perf_check or fi_test or fi_test_dfuse:
+        fs = subprocess.run([os.path.join(conf['PREFIX'], 'bin', 'fault_status')], check=False)
+        print(fs)
+        if fs.returncode == 0:
+            run_fi = True
+        else:
+            print("Unable to verify fault injection feature")
```

Comment:
```suggestion
            print("Unable to verify fault injection feature on release builds")
```

Just so the log reader knows why it was unable and that it's an expected situation.

Reply:
There are other potential reasons for fault injection to be compiled out.